libgoal/libgoal.go

```diff
@@ -1023,6 +1023,9 @@ func MakeDryrunStateGenerated(client Client, txnOrStxn interface{}, other []tran
 	for _, txn := range txns {
 		tx := txn.Txn
 		if tx.Type == protocol.ApplicationCallTx {
+			accounts := append(tx.Accounts, tx.Sender)
+			accounts = append(accounts, otherAccts...)
+
 			apps := []basics.AppIndex{tx.ApplicationID}
 			apps = append(apps, tx.ForeignApps...)
```

Comment:
Not directly relevant to this change, but should this code also collect all foreign assets and look up the account balances of their creator to add to `dr.Accounts`?